src/borg/testsuite/archiver/create_cmd.py

```diff
@@ -626,6 +627,46 @@ def test_file_status_excluded(self):
         if has_lchflags:
             self.assert_in("x input/file3", output)
 
+    def test_file_status_counters(self):
+        """Test file status counters in the stats of `borg create --stats`"""
+
+        def to_dict(borg_create_output):
+            borg_create_output = borg_create_output.strip().splitlines()
+            borg_create_output = [line.split(":", 1) for line in borg_create_output]
+            borg_create_output = {
+                key: int(value)
+                for key, value in borg_create_output
+                if key in ("Added files", "Unchanged files", "Modified files")
+            }
+            return borg_create_output
+
+        # Test case set up: create a repository
+        self.cmd(f"--repo={self.repository_location}", "rcreate", RK_ENCRYPTION)
+        # Archive an empty dir
+        result = self.cmd(f"--repo={self.repository_location}", "create", "--stats", "test_archive", self.input_path)
+        result = to_dict(result)
+        assert result["Added files"] == 0
+        assert result["Unchanged files"] == 0
+        assert result["Modified files"] == 0
+        # Archive a dir with two added files
+        self.create_regular_file("testfile1", contents=b"test1")
+        time.sleep(1)  # testfile2 must have newer timestamps than testfile1
```

Comment:
can you try keeping the comment, but removing the "sleep" or making it much shorter?

there are only few fs with very rough granularity (e.g. FAT and similar) and we do not use them for testing.
most other fs nowadays have rather fine granularity, like us or ns.

a test run executes > 1000 automated tests and there should be only a sleep if really needed.